client/nivlheim_client

```diff
@@ -685,12 +702,14 @@ sub http_get($$) {
 	return $result;
 }
 
-sub http_post($$) {
+sub http_post($$$) {
 	# Yes, it would have been nice to use LWP for this,
 	# but I see no way of using my own function for
 	# opening an SSL connection that way.
 
-	my ($url, $postdataref) = @_;
+	my ($url, $postdataref, $use_client_cert) = @_;
+	print "POST $url\n" if ($opt{'debug'});
+	if (!defined($use_client_cert)) { $use_client_cert = 1; }
```

Comment:
Trenger vi denne? Eneste tilfellet hvor $use_client_cert ikke vil være definert er hvis noen kaller http_post med undef som siste argument.

Reply:
Jeg tenkte hvis man glemmer å sende med det 3. argumentet til http_post, men det er kanskje lite sannsynlig i og med at sub'en er definert øverst i scriptet så Perl klager hvis man angir feil antall argumenter.
For øvrig trenger scriptet å gjøre http post uten klientsertifikat mot `/cgi-bin/reqcert`, mens den skal gjøre post _med_ sertifikat mot `/cgi-bin/secure/post`. Så den subrutinen trenger den parameteren for å angi om den skal bruke klientsertifikat eller ikke.
På den andre siden hadde det muligens virket å kalle opp `reqcert` selv om man hadde brukt et  sertifikat. Det er mulig vi kunne ha droppet parameteren og bare alltid sendt med sertifikat hvis det fantes et.